src/assets/lima-config.yaml

```diff
@@ -17,8 +17,9 @@ provision:
     set -o errexit -o nounset -o xtrace
     errors=0
     mount --make-shared /
+    mount --make-shared /var/lib
     rmdir /tmp/.ICE-unix /tmp/.X11-unix || true
```

Comment:
This is not really necessary. It doesn't hurt either, but one may wonder why we bother.

Reply:
Right; fixed in 2bd2392